### ISSUE TITLE
Measurements of the four weight sensors on the Nintendo Balance board are exactly 4 times too big

### ISSUE THREAD

**Issue by lshachar** [BrianPeek/WiimoteLib#6]
[Back in December of 2007, Mr Hiroyuki Ogasawar (Oga) from Japan](https://wlog.flatlib.jp/index.php?virtualpath=archive/1/2007-12-16), hacked the Wii Balance board, and figured out the formula to get weight values from the 4 weight sensors.

He posted the formula for the calculation, and wrote: (Google translate)
"I calculated this for 4 sensors of A, B, C, and D and took the average,
it matched my weight by itself."

...emphasis on the word "average".
Each weight sensor should return the proper portion of the person's weight that is currently standing on the device...
and the entire person weight will be given by *the total* of the 4 sensor weight values, *not* by averaging them!
I.E, a person that is standing so that his balance is a little to the heels of his left foot: BottomLeft:30kg , TopLeft:20kg, BottomRight:10kg, Top Right:5kg.
This guy weighs 30+20+10+5 = 65kg,
This guy does not weigh (30+20+10+5)/4 = 16.25kg!

The bug can be seen with the wiimoteTest program, and also with willBalanceWalker program, etc. all of the 4 sensor values are exactly 4 time too big. and the library divides the overall weight value by 4 (averaging), that's why the overall weight is still correct.

The bug was created when Oga used 68 (kg) in his formulas, instead of 17. I don't know how or who figured out that Nintendo used 17kg and 34kg weights as the calibration values on the wii balance board, but these are the correct numbers (17 / 34), wheres 68 is 4 times 17, or 4 times too big, hence explaining the error.
(If anyone has any internal nintendo documents that they would like to share with me... It might answer some serious questions I got, posted in #8  ) 

(please see Pull request #7 )

**Comment by lshachar** [comment]
I released my updated library [here](https://github.com/lshachar/WiimoteLib/releases/tag/v1.7.0.1)

**Comment by ZaqTheDuck** [comment]
Sorry for the offtop, but I have some trouble using these tools and you seem to understand this tool... So I have a Wiimote & Wiimoteplus and trying to connect a nunchuck. When I do that, the testing app crashes and later it just can't start. Have you faced anything like that?

**Comment by lshachar** [comment]
nope, as I'm not using any other wii remote except the balance board. (and it sounds like you're not using it)

**Comment by ZaqTheDuck** [comment]
I intended to do something with the balance board later (I have it) but currently I can't even fully set up the remote :(
